common/src/main/java/com/sequenceiq/cloudbreak/common/type/TemporalStorage.java

```diff
@@ -0,0 +1,5 @@
+package com.sequenceiq.cloudbreak.common.type;
+
+public enum TemporalStorage {
```

Comment:
Did not you rather mean `temporary`? "Temporal" means "time-dependent" or "related to time". 😄

Reply:
Fair point, thanks :)